src/views/Amalgamation/BusinessInfo.vue

```diff
@@ -1,28 +1,54 @@
 <template>
-  <div id="amalgamation-regular-business-info">
+  <div id="amalgamation-business-info">
     <!-- Registered Office Addresses -->
     <section
       v-show="isEntityType"
       class="mt-10"
     >
       <header id="office-address-header">
-        <h2>
-          Registered and Records Office Addresses
-        </h2>
-        <p>
+        <h2>Registered and Records Office Addresses</h2>
+        <p v-if="isAmalgamationFilingRegular">
           Enter the Registered Office and Records Office Mailing and Delivery Addresses of the Resulting
           businesses. All addresses must be located in BC.
         </p>
+        <p v-if="isAmalgamationFilingHorizontal || isAmalgamationFilingVertical">
+          The resulting business will adopt the following Registered Office and Records Office Mailing
+          and Delivery Addresses of the holding / primary company in this amalgamation.
+        </p>
```

Comment:
If this component was generic then I'd store the strings in the resource files, but this component is specific to amalgamations so I decided to just do this.